### DISCUSSION TITLE
Changing access token

### DISCUSSION THREAD

**Discussion by Jerk-Solutions** [ghostfolio/ghostfolio#1653] [Q&A]
In the self hosted version, is there any way to change access token once the account has been created? 
Assuming full access to the DB and the app is running inside docker

**Comment by dtslvr** [comment]
Hi @Jerk-Solutions

You should be able to create a new user. Then copy the `accessToken` of this new user from the `User` table to your original user.
Make sure to change the `accessToken` from the new user that you don't have two of the same.
